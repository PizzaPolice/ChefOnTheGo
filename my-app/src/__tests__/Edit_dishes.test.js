import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, mount, render } from 'enzyme';
import EditDishes from '../components/edit_dishes.jsx'


it('EditDishes component renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<EditDishes />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

// This test is shallow rendering our EditDishes component
describe('EditDishes Component', () => {
    it('EditDishes should render without throwing an error', () => {
        expect(shallow(<EditDishes />).exists(<form className='EditDishes'></form>)).toBe(true)
      })
    });