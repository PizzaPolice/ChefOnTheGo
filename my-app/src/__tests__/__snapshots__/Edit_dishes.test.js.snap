// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditDishes Component should render initial layout 1`] = `
Array [
  <div>
    <title>
        Edit Dishes
    </title>
    <h1>
        Create a new dish!
    </h1>
    <form
        className="new_dish"
    >
        <h3>
            Dish name
        </h3>
        <input
            id="dish_name"
            type="text"
        />
        <h3>
            Price
        </h3>
        <input
            id="dish_price"
            type="number"
        />
        <h3>
            Description
        </h3>
        <textarea
            cols="70"
            defaultValue="Enter a description of your dish"
            id="dish_desc"
            rows="10"
        />
        <input
            onClick={[Function]}
            type="button"
            value="Create your dish!"
        />
    </form>
    <h1>
        Edit your offerings
    </h1>
    <h2>
        Check the checkbox below a dish to select it for deletion
    </h2>
    <input
        onClick={[Function]}
        type="button"
        value="Delete selected dishes"
    />
    <div
        id="current_dishes"
    />
</div>,
]
`;
